src/components/Other/Project.vue

```diff
@@ -5,26 +5,26 @@
         <v-card>
           <v-card-title>
             <div>
-              <h2 class="primary--text">{{ project.name }}</h2>
-              <div class="primary--text">{{ project.team }}</div>
+              <h2 class="primary--text">{{ project[0].name }}</h2>
```

Comment:
Not good. Create a getter which returns a project by ID, and if that getter returns null, then fetch from firebase and store in store